woocommerce/wc-admin
# Adds SCSS color variables and breakpoints

Closes #27.

This PR imports a couple color variables to start with, and the breakpoint mixin from Calypso. It also moves a few wp-admin / reset rules out of the dashboard CSS and into it's own file. Webpack is used to automatically import these, so they can be used in your .scss files.

To Test:
* Build the files and visit the dashboard. Verify everything looks OK.
* To test the breakpoint, you can add something like the below to `.woo-dashboard` in dashboard/style.scss and see the background at certain screen sizes:

```
@include breakpoint( '>480px' ) {
	background: #000;
}
```
